### ISSUE TITLE
Installing more than the 12 fonts listed

### ISSUE THREAD

**Issue by jay-ramani** [mistu01/mffmv11#2]
I have 23 fonts (from FortyThreeCondensed) that are named
```
GoogleSans-Bold.ttf
GoogleSans-BoldItalic.ttf
GoogleSans-Medium.ttf
GoogleSans-MediumItalic.ttf
GoogleSans-Regular.ttf
Roboto-Black.ttf
Roboto-BlackItalic.ttf
Roboto-Bold.ttf
Roboto-BoldItalic.ttf
Roboto-Italic.ttf
Roboto-Light.ttf
Roboto-LightItalic.ttf
Roboto-Medium.ttf
Roboto-MediumItalic.ttf
Roboto-Regular.ttf
Roboto-Thin.ttf
Roboto-ThinItalic.ttf
RobotoCondensed-Bold.ttf
RobotoCondensed-BoldItalic.ttf
RobotoCondensed-Italic.ttf
RobotoCondensed-Light.ttf
RobotoCondensed-LightItalic.ttf
RobotoCondensed-Regular.ttf
```
While the project documentation says we're going to need 12 font styles to use this template
```
Black.ttf
BlackItalic.ttf
Bold.ttf
BoldItalic.ttf
Medium.ttf
MediumItalic.ttf
Regular.ttf
Italic.ttf
Light.ttf
LightItalic.ttf
Thin.ttf
ThinItalic.ttf
```
, how can I make use of all the 23 fonts listed? I'm a big fan of condensed fonts, and would very much like all of them to be used.

Apologies for raising this as an issue, but the repository doesn't have a Discussion tab.

Appreciate any help. Thanks and cheers.

**Comment by jay-ramani** [comment]
> I have 23 fonts (from FortyThreeCondensed) that are named
> 
> ```
> GoogleSans-Bold.ttf
> GoogleSans-BoldItalic.ttf
> GoogleSans-Medium.ttf
> GoogleSans-MediumItalic.ttf
> GoogleSans-Regular.ttf
> Roboto-Black.ttf
> Roboto-BlackItalic.ttf
> Roboto-Bold.ttf
> Roboto-BoldItalic.ttf
> Roboto-Italic.ttf
> Roboto-Light.ttf
> Roboto-LightItalic.ttf
> Roboto-Medium.ttf
> Roboto-MediumItalic.ttf
> Roboto-Regular.ttf
> Roboto-Thin.ttf
> Roboto-ThinItalic.ttf
> RobotoCondensed-Bold.ttf
> RobotoCondensed-BoldItalic.ttf
> RobotoCondensed-Italic.ttf
> RobotoCondensed-Light.ttf
> RobotoCondensed-LightItalic.ttf
> RobotoCondensed-Regular.ttf
> ```
> 
> While the project documentation says we're going to need 12 font styles to use this template
> 
> ```
> Black.ttf
> BlackItalic.ttf
> Bold.ttf
> BoldItalic.ttf
> Medium.ttf
> MediumItalic.ttf
> Regular.ttf
> Italic.ttf
> Light.ttf
> LightItalic.ttf
> Thin.ttf
> ThinItalic.ttf
> ```
> 
> , how can I make use of all the 23 fonts listed? I'm a big fan of condensed fonts, and would very much like all of them to be used.
> 
> Apologies for raising this as an issue, but the repository doesn't have a Discussion tab.
> 
> Appreciate any help. Thanks and cheers.

@mistu01?

**Comment by mistu01** [comment]
@jay-ramani you should use [CFI](https://github.com/nongthaihoang/custom_font_installer) then, mffm is pretty limited.

**Comment by jay-ramani** [comment]
Yup, my issue was posted on 11th December. Have switched over to CFI since. CLosing the issue.

**Comment by jay-ramani** [comment]
Yup, my issue was posted on 11th December. Have switched over to CFI since. CLosing the issue.
